src/schema/alter-column-builder.ts

```diff
@@ -66,20 +56,30 @@ export class AlterColumnBuilder {
 }
 
 /**
- * Allows us to force consumers to do something, anything, when altering a column.
+ * Allows us to force consumers to do something, anything, when altering a column,
+ * and also disallows chaining more methods on AlterColumnBuilder, because
+ * SQL syntax for chaining ALTER COLUMN operations would be not WYSIWYG
```

Comment:
I think the addition is TMI.

```suggestion
 * Allows us to force consumers to do exactly one alteration to a column.
```